File: Buoi1/Big-omega/timkiemkhongsapxep.jl
function find_first_element(arr, target)
    if arr[1] == target  # Chỉ số trong Julia bắt đầu từ 1
        return 1
    end
    return -1
end

arr = [10, 20, 30, 40, 50]
println(find_first_element(arr, 10))  # Output: 1

File: Buoi1/Big-omega/timkiemkhongsapxep.cpp
#include <iostream>
#include <vector>
using namespace std;

int find_first_element(const vector<int>& arr, int target) {
    if (arr[0] == target) {
        return 0;
    }
    return -1;
}

int main() {
    vector<int> arr = {10, 20, 30, 40, 50};
    cout << find_first_element(arr, 10) << endl;  // Output: 0
    return 0;
}
